client/me/purchases/purchases-list/index.jsx

```diff
@@ -69,6 +70,15 @@ class PurchasesList extends Component {
 		}
 
 		if ( this.props.hasLoadedUserPurchasesFromServer && ! this.props.purchases.length ) {
+			if ( this.props.hasMoreThanOneSite ) {
```

Comment:
Aren't you checking the wrong thing here?  I think we actually want this to check if the user has no sites, not if the user has more than one site.

Reply:
🤦‍♂️ yes, I am. Thank you. I'll fix that.